app/addons/documents/header/header.react.jsx

```diff
@@ -43,7 +43,7 @@ function (app, FauxtonAPI, React, Actions, ReactComponents,
         selectedView: indexResultsStore.getCurrentViewType(),
         isTableView: indexResultsStore.getIsTableView(),
         includeDocs: queryOptionsStore.getIncludeDocsEnabled(),
-        bulkDocCollection: indexResultsStore.getBulkDocCollection(),
+        bulkDocCollection: indexResultsStore.getBulkDocCollection()
```

Comment:
note: commas at the end are not harmful, they were back in IE6 or IE8 ages which was not acting according to the ES spec. leaving the commas makes it easier to add things in the future